pyproject.toml

```diff
@@ -71,11 +71,27 @@ testpaths = "assesspy/tests"
 legacy_tox_ini = """
 [tox]
 min_version = 4.0
-envlist = py39, py310, py311, py312, py313
+envlist =
+    py{39, 310}-lowest
+    py{310, 311, 312, 313}
+
+[gh-actions]
+python =
+    3.9: py39
+    3.10: py310
+    3.11: py311
+    3.12: py312
+    3.13: py313
 
 [testenv]
-deps =
-    setuptools>=61.0
+extras = dev
+commands = pytest
+
+[testenv:py{39, 310}-lowest]
+uv_resolution = lowest-direct
+
+[testenv:py{39, 310, 311, 312, 313}]
+uv_resolution = highest
```

Comment:
This isn't strictly necessary (`uv_resolation = highest` is the default), but I figure it helps to be explicit.

Reply:
**[Question, non-blocking]** It seems like this config is supposed to run two sets of tests for Python 3.9 and 3.10, but when I look at the [build logs](https://github.com/ccao-data/assesspy/actions/runs/12059340171/job/33627775548#step:6:25), it seems like only the `-lowest` versions of 3.9 and 3.10 are running. Is that right, or am I misreading the logs?

Reply:
It's a little confusing, but per the `envlist` section above, there actually is no `py39`. Only `py310-lowest` and `py310` overlap in the env list, so the 3.10 runner is the only one that runs two sets of tests. The config here just specifies that if `py39` were to exist, it would use this option. I'll update it though to remove `py39` and make it a bit clearer.

Reply:
I see now, thanks!